.github/workflows/linked-issue.yml

```diff
@@ -0,0 +1,23 @@
+name: Verify Linked Issue
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+jobs:
+  verify-linked-issue:
+    runs-on: ubuntu-latest
+    if: github.actor != 'dependabot[bot]' && github.actor != 'dependabot-preview[bot]'
+    name: Verify Pull Request references Issue
+    steps:
+      - uses: Codeinwp/verify-linked-issue-action@6da9af91d78847c7ea264083a64eab3271b4369f
```

Comment:
Any particular reason for using this fork?

Reply:
I found the fork through this PR on the original repo:

https://github.com/hattan/verify-linked-issue-action/pull/30

In particular, the fork shouldn't suffer the known issues of the original.

Reply:
Oh nice. That includes the regex issue with the required trailing space? Either way I'm fine with giving it a try.

Reply:
Yea, this PR itself has the action running on it and it successfully detected the two issues I referenced, neither of which were written with a trailing space.